src/pchevrel/Verif.php

```diff
@@ -275,6 +275,34 @@ public function isEqualTo($string)
     }
 
     /**
+     * Check if the remote content fetched contains a string
+     * @param  string $string The string we expect
+     * @return $this
+     */
+    public function contains($string)
+    {
+        $test = 0;
+        foreach (explode (PHP_EOL, $this->content) as $line){
+            if (strpos ($line, $string)) {
+                $test = 1;
+            }
+        }
+        if (! $test) {
+            $this->errors[] =
+                $this->colorizeOutput('Not present: ', 'yellow')
+                . $this->colorizeOutput($this->uri, 'blue')
+                . "\n"
+                . $this->colorizeOutput('* Expected: ', 'green') . $string
+                . "\n"
+                . $this->colorizeOutput('* Received: ', 'red') . $this->content
+                . "\n";
+        }
```

Comment:
You don't need to transform $this->content from  a string to an array and then loop through all array items as a string. You just need to check if your $string is in $this->content like this:
```php
    public function contains($string)
    {
        if (strpos($this->content, $string) === false) {
            $this->errors[] = $this->colorizeOutput('Missing content: ', 'red') . $string;
        }

        $this->test_count++;

        return $this;
    }
``` 
Note that we are using a strict equlity (===) because if your string is at position 0, then it would be evaluated to false.

You also don't need to output $this-content in the string as we are not expecting a specicif content but are in a true/false situation.